src/components/TimeCardPage/CellTypes/CommentModals/ShowReportModal.tsx

```diff
@@ -169,10 +173,36 @@ export default function ShowReportModal({
         setReports([...reports, createNewReport(user, remark, selectedTime.unix())]);
         console.log(createNewReport(user, remark, parseInt(selectedTime.format('X')))); // currently gmt TODO: fix later
       }
-
-      // show a toast that its been submitted
+      apiClient.saveReport(remark, timesheetID).then((resp) =>
+          {if (resp) { 
+              toast({
```

Comment:
This might be overkill... but just to keep things stylistically the same for everyone, if we think we going to need toast notifications for a bunch of things, maybe put the toast into a wrapper function and just pass in the status and description? Just so it's easier for people to create a toast notification and make sure it's the same style as this in the future. Also will make it easier if we want to change the look of the toast notifications in the future (i.e. change position, add in more styling, etc.)

Something like:
`def createToast(status: string, description: string`)...`